plugins/outputs/whatap/whatap_test.go

```diff
@@ -0,0 +1,80 @@
+package whatap
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"testing"
+
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/testutil"
+
+	whatap_hash "github.com/whatap/go-api/common/util/hash"
+
+	"github.com/stretchr/testify/require"
+)
+
+func newWhatap() *Whatap {
+	hostname, _ := os.Hostname()
+	return &Whatap{
+		Timeout: config.Duration(60 * time.Second),
+		Log:     testutil.Logger{},
+		oname:   hostname,
+		oid:     whatap_hash.HashStr(hostname),
+	}
+}
+func TestWhatapConnect(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
```

Comment:
Why can't this be executed in short-mode? As long as you do not leave localhost this should be ok.

Reply:
Deleted that phrase